CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,127 @@
+# Contributor Covenant Code of Conduct
```

Comment:
For our future reference: the original source of this is https://github.com/r-lib/usethis/blob/main/inst/templates/CODE_OF_CONDUCT.md